eventually/authentication/models.py

```diff
@@ -79,11 +111,11 @@ def create(first_name, middle_name, last_name, email, password):
                           password=password)
         try:
             user.save()
-        except CustomUser.OperationalError():
-            pass
-        return user
+            return user
+        except django.db.utils.IntegrityError:
```

Comment:
maybe it makes sense to import only `IntegrityError` and to use it without `django.db.utils`?

Reply:
Fixed.